Sources/LicensePlistCore/Extension/APIKit.extension.swift

```diff
@@ -19,4 +19,17 @@ extension LicensePlistExtension where Base: Session {
             return self.sendSync(request)
         }
     }
+
+    static var shared: Session {
+        Session(adapter: GitHubURLSessionAdapter(configuration: .default))
+    }
+}
+
+final class GitHubURLSessionAdapter: URLSessionAdapter {
+    // This method is to assist with persistence of an authorization token incase of a redirect.
+    func urlSession(_ session: URLSession, task: URLSessionTask, willPerformHTTPRedirection response: HTTPURLResponse, newRequest request: URLRequest) async -> URLRequest? {
+        var request = request
+        request.setValue(task.originalRequest?.value(forHTTPHeaderField: "Authorization"), forHTTPHeaderField: "Authorization")
```

Comment:
For security I think we should check the original requests host matches the new request host, otherwise, log a message stating the token wasn't forwarded on.

We may have to include additional GitHub domains if they need authentication as well.

Just think may want to err on the side of caution here.